### ISSUE TITLE
Add softmax cross entropy loss to dlfs_kuoste package

### ISSUE THREAD

**Issue by Kuoste** [Kuoste/DLFS_code#1]
fyi
there's a nice explanantion on how the derivative of the softmax cross entropy loss is derived in the math slides [04_extensions/Math.ipynb](https://github.com/Kuoste/DLFS_code/blob/master/04_extensions/Math.ipynb)
